src/kit/shell/src/shellEngine.c

```diff
@@ -1241,7 +1241,7 @@ int wsclient_send(char *strdata, WebSocketFrameType frame) {
     payload_offset += 8;
   } else {
     fprintf(stderr, "websocket send too large data\n");
-    return -1;
+    return 1;
```

Comment:
What's the difference between 1 and -1? where it be checked?

Reply:
There is no difference, just make it the same as the error return in line 1277.

Reply:
Return a minus number is more industry practice.